datahub-upgrade/src/main/java/com/linkedin/datahub/upgrade/common/steps/ClearGraphServiceStep.java

```diff
@@ -0,0 +1,58 @@
+package com.linkedin.datahub.upgrade.common.steps;
+
+import com.linkedin.datahub.upgrade.UpgradeContext;
+import com.linkedin.datahub.upgrade.UpgradeStep;
+import com.linkedin.datahub.upgrade.UpgradeStepResult;
+import com.linkedin.datahub.upgrade.impl.DefaultUpgradeStepResult;
+import com.linkedin.datahub.upgrade.nocode.NoCodeUpgrade;
+import com.linkedin.metadata.graph.GraphService;
+import java.util.function.Function;
+
+
+public class ClearGraphServiceStep implements UpgradeStep {
+
+  private final String deletePattern = ".*";
+
+  private final GraphService _graphService;
+  private final boolean _alwaysRun;
+
+  public ClearGraphServiceStep(final GraphService graphService, final boolean alwaysRun) {
+    _graphService = graphService;
+    _alwaysRun = alwaysRun;
+  }
+
+  @Override
+  public String id() {
+    return "ClearGraphServiceStep";
+  }
+
+  @Override
+  public boolean skip(UpgradeContext context) {
+    if (_alwaysRun) {
+      return false;
+    }
+    if (context.parsedArgs().containsKey(NoCodeUpgrade.CLEAN_ARG_NAME)) {
```

Comment:
Should we add a "cleanup has been requested" report line here?

Reply:
That log is added in the execute function. I can move it to here?

Reply:
no lets just keep that